projects/core/src/user/store/effects/user-replenishment-orders.effect.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { Actions } from '@ngrx/effects';
+import { provideMockActions } from '@ngrx/effects/testing';
+import { cold, hot } from 'jasmine-marbles';
+import { Observable, of, throwError } from 'rxjs';
+import { ReplenishmentOrderList } from '../../../model/replenishment-order.model';
+import { UserReplenishmentOrderAdapter } from '../../connectors/replenishment-order/user-replenishment-order.adapter';
```

Comment:
with line 9, you can point to /index in order to put adapter and connector in the same import line